data_processors/pipeline/tools/liborca.py

```diff
@@ -173,7 +173,12 @@ def parse_umccrise_workflow_output_directory(output_json: str, deep_check: bool
     :return:
     """
 
-    lookup_keys = ['main/umccrise_output_directory', 'umccrise_output_directory']
+    lookup_keys = [
+        # Old keys < 2023.01
+        'main/umccrise_output_directory', 'output_directory'
```

Comment:
Ah. Almost. Typo here. 🙂 And need comma.

```
'main/umccrise_output_directory', 'umccrise_output_directory',
```

Reply:
🤦

Reply:
Resolved in 250e23a6968abc0b4c24580e93fa1e5f030903c2